src/main/java/com/webperside/courseerpbackend/models/mappers/CountryEntityMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.webperside.courseerpbackend.models.mappers;
+
+import com.webperside.courseerpbackend.models.mybatis.country.Country;
+import com.webperside.courseerpbackend.models.payload.country.CountryPayload;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface CountryEntityMapper {
+    CountryEntityMapper INSTANCE = Mappers.getMapper(CountryEntityMapper.class);
+
+
+    Country toEntity(CountryPayload countryPayload);
+    Country toEntity(CountryPayload countryPayload, @MappingTarget Country country);
```

Comment:
remove that method. we'll map our payload model to mybatis model and send it to service directly

Reply:
Do you mean to remove toEntity? So how would I use countryPayload without mapping?